packages/salesforcedx-vscode-soql/package.json

```diff
@@ -10,15 +10,27 @@
     "url": "https://github.com/forcedotcom/salesforcedx-vscode"
   },
   "version": "50.5.0",
+  "preview": true,
```

Comment:
This will mark our extension as a preview (beta) in the marketplace.

More info can be found [here](https://code.visualstudio.com/api/references/extension-manifest)